MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.c

```diff
@@ -166,6 +166,15 @@ EnumerateNvmeDevNamespace (
     Device->BlockIo2.FlushBlocksEx = NvmeBlockIoFlushBlocksEx;
     InitializeListHead (&Device->AsyncQueue);
 
+    //
+    // Create Media Sanitize Protocol instance
+    //
+    Device->MediaSanitize.Revision       = EFI_MEDIA_SANITIZE_PROTOCOL_REVISION;
+    Device->MediaSanitize.Media          = &Device->Media;
+    Device->MediaSanitize.SanitizeAction = 0;

```

Comment:
why zero?

Reply:
Zero indicates to action to take for sanitize. Similar to setting a status to 0. 

Do you have a recommendation on another default beginning value and protocol install time?

Reply:
Resolved by adding #define for No Action (zero). Will also convert Sanitize Action to Sanitize Capabilities.